ironfish/src/primitives/transaction.test.slow.ts

```diff
@@ -46,6 +48,46 @@ describe('Accounts', () => {
     expect(transactionB.isMinersFee()).toBe(true)
   })
 
+  it('throw error if account is not fully synced when creating transaction', async () => {
+    const nodeA = nodeTest.node
+
+    // Create an account A
+    const accountA = await useAccountFixture(nodeA.accounts, () =>
```

Comment:
nit: seems like pattern is usually: 
```
const accountA = await useAccountFixture(accounts, 'accountA')
const accountB = await useAccountFixture(accounts, 'accountB')
```
using the string instead of generator function